tests/testTriggers.go

```diff
@@ -1066,11 +1066,15 @@ func haIncreaseWithErrorInjection(event *EventRecord, contexts *[]*scheduler.Con
 						}
 					}
 				}
+				log.Debugf("Printing the volume inspect for the volume:%s ,volID:%s and namespace:%s", v.Name, v.ID, v.Namespace)
+				PrintInspectVolume(v.ID)
 
 				if err == nil {
 					HaIncreaseRebootTargetNode(event, selctx, v, storageNodeMap, errorInj)
 					HaIncreaseRebootSourceNode(event, selctx, v, storageNodeMap, errorInj)
 				}
+				log.Debugf("Printing the volume inspect for the volume:%s ,volID:%s and namespace:%s after HA increase and reboot of the source node ", v.Name, v.ID, v.Namespace)
```

Comment:
add this after operation on target and also after source node

Reply:
added